bg_mon.c

```diff
@@ -358,6 +359,16 @@ static struct evbuffer *prepare_statistics_output(struct timeval time, system_st
 	evbuffer_add_printf(evb, "{\"hostname\":\"%s\",\"time\":%llu,\"sysname\":\"Linux: %s\",", s.hostname, ts, s.sysname);
 	evbuffer_add_printf(evb, "\"cpu_cores\":%d,\"postgresql\":{\"version\":\"%s\"", c.cpu_count, PG_VERSION);
 	evbuffer_add_printf(evb, ",\"role\":\"%s\",", p.recovery_in_progress?"replica":"master");
+
+	if (p.recovery_in_progress) {
+		evbuffer_add_printf(evb, "\"wal\":{\"replay_paused\":%s,", w.is_wal_replay_paused?"true":"false");
```

Comment:
The `\"wal\":{` block could go to the [previous line](https://github.com/CyberDem0n/bg_mon/pull/39/files#diff-57f1b76e196394f418a39a1f6d499e8da40bc02a013bf7db13a092fa79c1a60aR361).
The same applies to the closing curly brace, which could go to the line https://github.com/CyberDem0n/bg_mon/pull/39/files#diff-57f1b76e196394f418a39a1f6d499e8da40bc02a013bf7db13a092fa79c1a60aR372.